### ISSUE TITLE
suscription need?

### ISSUE THREAD

**Issue by kiloptero** [lukas-clarke/eight_sleep#17]
hi all! thanks for this!! you need the suscription for this?

**Comment by lukas-clarke** [comment]
I believe you can use this integration without a subscription. This integration uses the same calls as the app. And I believe the app allows the calls the integration is using, even when not having a subscription.
I don't have any way to test it out though, because I have a grandfathered account.
And this isn't saying anything about what may happen in the future. Eight Sleep may change the API at any point.

I have made a note to add this info to the readme for the next release.

**Comment by kiloptero** [comment]
yes... the hw is amazing.... but dont trust the business model.. almost buy this things hours ago until reaad all the bad comments

**Comment by eugenk** [comment]
My subscription (Pod 3) has expired some time ago and I just tested this integration. These features are working fine for me:

- Sensor `${name}_s_bed_state` (current temperature of the water - ranged from -100 to +100, representing the range -10 to 10 in the app's temperature dial)
- Service `eight_sleep.side_on` (turn on the water temperature control for one side of the bed)
- Service `eight_sleep.side_off` (turn off the water temperature control for one side of the bed)
- Service `eight_sleep.heat_increment` (increase/decrease the temperature of a side by a value that you choose)
- Service `eight_sleep.heat_set` (set the temperature of a side to a value between -100 and 100)

Actually these are exactly the features I used with the included subscription.

Alarms cannot be set up, which is totally fine for me.
